wandb/apis/public.py

```diff
@@ -4700,26 +4704,148 @@ def save(self):
         }
         """
         )
+        introspect_query = gql(
+            """
+            query ProbeServerAddAliasesInput {
+               AddAliasesInputInfoType: __type(name: "AddAliasesInput") {
+                   name
+                   inputFields {
+                       name
+                   }
+                }
+            }
+            """
+        )
+        res = self.client.execute(introspect_query)
+        valid = res.get("AddAliasesInputInfoType") or None
+        aliases = None
+        if not valid:
+            # Note: dangerous for artifacts linked to multiple collections
```

Comment:
Can we be explicit here in the comment. for example: 
```
For Core Versions < X, this is always a safe call
For Core Versions between X and Y, this call may inadvertently delete aliases when the artifact is a member of more than one collection
For Core Versions between Y and Z, this call will result in a gorilla error when the artifact is a member of more than one collection
For Core Versions > Z, we will use the explicit add/remove aliases mutation - which supports multi-membership
```

Reply:
Verified that this risk doesn't exist. All we need to check for is the existence of the new alias endpoints. The ability to create portfolios in the UI came afterwards. 

I added a comment explaining the version check.